ui/app/components/secret-engine/configurable-secret-engine-details.hbs

```diff
@@ -0,0 +1,42 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#if this.configError}}
+  {{! Surface API errors not associated with empty configuration details }}
+  <EmptyState data-test-config-error={{@model.id}} @title="Something went wrong" @message={{this.configError}} />
```

Comment:
Should we use the `Page::Error` component here instead? That way the empty state messages are consistent

https://github.com/hashicorp/vault/blob/4ac07e1d97236af2ff675d2863a5acb010f3eaff/ui/lib/ldap/addon/components/page/configuration.hbs#L38-L40

Reply:
🫠 I debated, and personally I like the look better of the empty state. But yes, I suppose you're right.